modules/javafx.media/src/main/native/jfxmedia/platform/osx/OSXMediaPlayer.mm

```diff
@@ -327,6 +327,7 @@ - (void) stop
     if ([scheme caseInsensitiveCompare:@"jar"] == NSOrderedSame ||
         [scheme caseInsensitiveCompare:@"jrt"] == NSOrderedSame) {
         CJavaInputStreamCallbacks *callbacks = new (nothrow) CJavaInputStreamCallbacks();
+        jobject jConnectionHolder = CLocator::CreateConnectionHolder(env, jLocator);
         if (callbacks == NULL) {
```

Comment:
Missing `NULL` check for  `jConnectionHolder`. `if (callbacks == NULL || jConnectionHolder == NULL)`

If `callbacks` is not `NULL` it needs to be deleted.